components/externalauth/android/java/src/org/chromium/components/externalauth/ExternalAuthUtils.java

```diff
@@ -259,7 +259,10 @@ protected int checkGooglePlayServicesAvailable(final Context context) {
         // TODO(crbug.com/577190): Temporarily allowing disk access until more permanent fix is in.
         try (StrictModeContext ignored = StrictModeContext.allowDiskWrites();
                 TraceEvent e = TraceEvent.scoped("checkGooglePlayServicesAvailable")) {
-            return ChromiumPlayServicesAvailability.getGooglePlayServicesConnectionResult(context);
+            // TODO(wolvic): Use ChromiumPlayServicesAvailability once wolvic
+            // can properly depend on GMS services.
+            // return ChromiumPlayServicesAvailability.getGooglePlayServicesConnectionResult(context);
+            return 0;
```

Comment:
Yes, I created a bug here https://github.com/Igalia/wolvic-chromium/issues/102. I tried to spend some time on this but after including Google Play Services dependency I ran into some missing AndroidManifest entries, I suspect that I didn't add the dependency correctly. I don't quite understand how it's done in upstream since they use GN builds instead of gradle and it seems to be implicitly included somehow.